### PR TITLE
feat!: update typeorm entities and endpoint with the new updated design v57

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✔                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: src/task/models/tasksManager.ts [hunk 1]
```diff
@@ -32,135 +35,153 @@ export class TasksManager {
...
+      customerName,
+    });
     const task = await this.taskRepository.getCustomerTaskByJobId(jobId, customerName);
 
-    if (task) {
-      this.logger.info({ msg: `found similar task id ${task.id} with job id ${jobId} and customer name: ${customerName}, updating task webhooks request`, jobId, customerName, webhooks: req.webhooks });
-      await this.webhooksRepository.upsertWebhooks(req.webhooks, task.id);
-      return task;
+    if (!task) {
+      const errMessage = `task with job id ${jobId} and customer name: ${customerName} was not found`;
```
Comment (asafMasa) [inline src/task/models/tasksManager.ts:120]: add to log "even though domain response declaired it's available"

File: src/task/models/tasksManager.ts [hunk 2]
```diff
@@ -32,135 +35,153 @@ export class TasksManager {
...
+      this.logger.error({
+        msg: errMessage,
+        jobId,
+        customerName,
+      });
+      throw new NotFoundError(errMessage);
```
Comment (asafMasa) [inline src/task/models/tasksManager.ts:126]: i don't think this is the response the client should get

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/task/models/tasksManager.ts [hunk 3]
```diff
@@ -74,16 +74,13 @@ export class TasksManager {
 
   public async getLatestTasksByLimit(limit: number): Promise<TaskResponse[]> {
     try {
-      this.logger.info({ msg: `get task by limit ${limit}`, limit });
+      this.logger.info({ msg: `get latest tasks by limit ${limit}`, limit });
       if (limit > this.maxTasksNumber) {
         throw new BadRequestError(`requested limit ${limit} is higher than the maximum possible limit tasks number ${this.maxTasksNumber}`);
       }
-      const tasksResponse: TaskResponse[] = [];
+      // const tasksResponse: TaskResponse[] = [];
```
Comment (asafMasa) [inline src/task/models/tasksManager.ts:81]: remove

File: src/task/models/tasksManager.ts (hunk 2 above)
Comment (asafMasa) [reply src/task/models/tasksManager.ts:123]: then lets talk with him...

Comment (CL-SHLOMIKONCHA) [reply src/task/models/tasksManager.ts:123]: ok

File: src/task/models/tasksManager.ts (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply src/task/models/tasksManager.ts:81]: removed

**Review by asafMasa — APPROVED** [review]

**Pull request merged**